papers/Raku_Steering_Council_Code.md

```diff
@@ -1,51 +1,36 @@
-council. The council has broad authority, which they seek to exercise
-as rarely as possible; instead, they use this power to establish
-standard processes.  It follows the general philosophy that it's better
-to split up large changes into a series of small changes that can be
-reviewed independently: instead of trying to do everything in one proposal,
-focus is on providing a minimal-but-solid foundation for further governance
-decisions.
-
-Rationale
-=========
-
-The main goals of this proposal are:
-
-* Be **boring**: We're not experts in governance, and we don't think
-Raku is a good place to experiment with new and untried governance
-models. So this proposal sticks to mature, well-known, previously
-tested processes as much as possible. The high-level approach of a
-mostly-hands-off council is arguably the most common across large
-successful F/OSS projects, and low-level details are derived
-directly from Python's governance, which was influenced by Django's
-governance.
-* Be **simple**: Pare things down to the minimum needed to make it
-workable: the council, the core team (who elect the council), and
-the process for changing the document. The goal is Minimum Viable
-Governance.
-* Be **comprehensive**: But for the things that need to be defined,
-it should cover all the bases.  Having a clear and unambiguous
-set of rules also helps minimize confusion and resentment.
-* Be **flexible and light-weight**: It will take time and experimentation
-to find the best processes for working together. By keeping this
-proposal as minimal as possible, maximal flexibility for adjusting
-things later is retained, while minimizing the need for heavy-weight
-and anxiety-provoking processes like whole-project votes.
+This document establishes the structure of governance for the Raku
+language and community.  Raku is governed by the Problem Solving
+process, the Core Team and the Raku Steering Council.  As set out
+below, the RSC has broad authority but seeks to exercise that
+authority as rarely as possible.
 
 Specification
 =============
 
 The Steering Council
 --------------------
 
-### Composition
+### Definitions 
+
+* **The Problem Solving process** – The consensus-driven process that
+  allows anyone to suggest, discuss, and agree to on major changes to
+  Raku.  The Problem Solving process is described at
+  [Raku/problem-solving](https://github.com/Raku/problem-solving) and
+  is intended to be the main way major Raku decisions are reached.
 
+* **The Core Team** – Contributors to the Raku language who are
+  trusted to make changes to major Raku repositories.  Criteria for
+  Core Team membership is described later in this document, and
+  current Core Team members are [listed
+  publicly](https://github.com/Raku/Raku-Steering-Council/blob/main/papers/Core_Team.md).
+  
+* **The Raku Foundation** – An alias for Yet Another Society, which is
+  also known by the alias The Perl Foundation (TPF).
```

Comment:
I agree to Geoffrey on this: it's better not to be too specific of what The Raku Foundation is. As a loose comparison I'd say this paper is a kind of Constitution. And as such it's better done in a way to minimize the number of edits needed whenever the situation changes.

Reply:
I disagree. If the raku foundation becomes a Different legal   organization, it’s worth an edit.